### ISSUE TITLE
[bug] Upload from Github Actions silently fails

### ISSUE THREAD

**Issue by anton-matosov** [conan-io/conan#6511]
I am trying to use GitHub Actions for CI of my packages and noticed that upload silently fails at `Compressing package...` step. 
Recipe is upload successfully thought

One of build runs that have "failed" upload https://github.com/Dr-QP/conan-arduino-sdk/runs/437882040

I have tried calling upload directly or using conan package tools with upload remote. It "skips/fails" upload on the compression stage.

However if I run it locally from terminal on macOS upload passes without any problems.

### Environment Details (include every applicable attribute)
  * Operating System+version: Windows, macOS, linux
  * Conan version: 1.22
  * Python version: 3.7

### Steps to reproduce (Include if Applicable)
Create github action workflow that installs conan package tools, builds package and tries to upload

### Logs (Executed commands with output) (Include/Attach if Applicable)
```

Uploading package 1/2: eea0e04e2f25bc4a83f5f147c227e20b35df8c5e to 'upload_repo'         

Checking package integrity...                                         
Package integrity OK!                                                 
Compressing package...

Uploading package 2/2: 2f653807fb9b655bbd2925db8d119ffeecb38e47 to 'upload_repo'         

Checking package integrity...                                         
Package integrity OK!                                                 
Compressing package...
```

**Comment by anton-matosov** [comment]
Seems to be dupe of https://github.com/conan-io/conan/issues/6482 
And my current workaround is to downgrade conan and CPT
```
pip install 'conan<1.21' 'conan_package_tools<0.31' --upgrade
```
With it the packages got uploaded successfully.

**Comment by memsharded** [comment]
Hi @anton-matosov 

yes, sorry about that. We are about to release 1.22.1 fixing this, and later we will backport fixes to 1.21.3 too. Thanks for reporting!
